src/hooks/useTotalStaked.ts

```diff
@@ -0,0 +1,51 @@
+import BN from 'bn.js'
+import { useQuery, gql } from '@apollo/client'
+import { useUsers } from 'store/cache/user/hooks'
+import getActiveStake from 'utils/activeStake'
+import { Status } from 'types'
+
+const GET_TOTAL_CLAIMABLE = gql`
+  query totalClaimable($id: String!) {
+    audiusNetwork(id: $id) {
+      totalTokensClaimable
+    }
+  }
+`
+
+interface TotalClaimable {
```

Comment:
just curious, why are we using interfaces rather than types?

Reply:
Oh I just copied the reference code and it used interfaces, no reason though.